tests/core/test_state_sync.py

```diff
@@ -372,16 +384,20 @@ def test_get_snapshot_intervals(
         ),
         version="c",
     )
-    state_sync.add_interval(snapshot_c, "2020-01-03", "2020-01-03")
     state_sync.push_snapshots([snapshot_c])
+    state_sync.add_interval(snapshot_c, "2020-01-03", "2020-01-03")
```

Comment:
Confirmed privately that moving this down a line is fine for the intent of the test.